script.adzapper/resources/language/English/strings.po

```diff
@@ -0,0 +1,155 @@
+# Kodi Media Center language file
```

Comment:
Please adopt the new language folder layout for krypton:

`resources/language/resource.language.en_gb/strings.po`